docs/user-docs/getting_started.rst

```diff
@@ -185,3 +185,7 @@ used as a feed for a second Pulp. Here is an example.::
    $ pulp-admin python repo create --repo-id sync_from_other_pulp --feed http://<host>/pulp/python/web/<repo_id>/ --package-names numpy,scipy
 
    $ pulp-admin python repo sync run --repo-id sync_from_other_pulp
+
+The pulp_python RPM distribution automatically adds index.json as a DirectoryIndex in pulp_python.conf. If you're not running the pulp_python RPM distribution, you will need to ensure this setting is present before remote Pulp instances will be able to sync your repository.
```

Comment:
I think this info would fit better https://github.com/mikeadamz/pulp_python/blob/0ac47451a77d5a65eb3827599ddc354c4d9ce490/docs/admin-docs/installation.rst#server